Makefile

```diff
@@ -36,6 +36,10 @@ endif
 prefix=/usr/local
 libdir=$(prefix)/lib
 includedir=$(prefix)/include
+pkgconfigdir=$(libdir)/pkgconfig
+
+pkglibdir=$(libdir:$(prefix)/%=%)
```

Comment:
I don't understand why you compute these variables. Why not use `libdir` directly in the pkgconfig file, and hardcode the prefix in each variable? I don't see the advantage of using `${prefix}` there.

Reply:
Here are some examples:

* https://people.freedesktop.org/~dbn/pkg-config-guide.html
* https://linux.die.net/man/1/pkg-config
* https://en.wikipedia.org/wiki/Pkg-config

Reply:
I see that, but what's the point when it makes the code more complex?

Reply:
When you run pkg-config, e.g. to get the appropriate cflags for compiling the library, you can use stuff like `pkg-config --define-prefix --cflags libpng`, which will change what is substituted for `$(prefix)` when it spits out the includedir.

Reply:
OK, makes sense then!